xml/operations-troubleshooting-recover_rabbit.xml

```diff
@@ -204,29 +204,68 @@ ardana-cp1-c1-m3-mgmt</screen>
    baremetal node, the &rabbit; cluster needs any references to the node
    removed from it. This is because &rabbit; associates a cookie with each node
    in the cluster which is derived, in part, by the specific hardware.
-   Replacing a hard drive in a node should be fine, but an exchanged
-   motherboard or replacing it with another node entirely will potentially
-   trigger a breakage. Under these circumstances, the running &rabbit; cluster
-   must be edited from a running &rabbit; node with these steps:
+   Replacing a hard drive in a node should be fine, but changing a motherboard
+   or replacing the node with another node entirely will potentially trigger a
+   breakage. Under these circumstances, the running &rabbit; cluster must be
+   edited from a running &rabbit; node.
+  </para>
+  <para>
+   In this example, controller 3 is the node being replaced with the following
+   steps:
   </para>
   <procedure>
+   <step>
+    <screen>&prompt.ardana;cd ~/scratch/ansible/next/ardana/ansible</screen>
+   </step>
    <step>
     <para>
      SSH to a running &rabbit; cluster node.
     </para>
+    <screen>&prompt.ardana;ssh cloud-cp1-rmq-mysql-m1-mgmt</screen>
+   </step>
+   <step>
+    <para>
+     Force the cluster to forget the node you are removing (in this example,
+     the controller 3 node).
+    </para>
+    <screen>&prompt.ardana;sudo rabbitmqctl forget_cluster_node \
+rabbit@cloud-cp1-rmq-mysql-m3-mgmt</screen>
+   </step>
+   <step>
+    <para>
+     Confirm that the node has been removed:
```

Comment:
_has been removed._ (period instead of colon)

Reply:
done